infrastructure/constants/auth.go

```diff
@@ -0,0 +1,15 @@
+package constants
+
+const (
+	AccessTokenCookie  = "access_token"
+	RefreshTokenCookie = "refresh_token"
+
+	AccessTokenDuration  = 15 * 60           // 15 minutes in seconds
+	RefreshTokenDuration = 30 * 24 * 60 * 60 // 30 days in seconds
+
+	CookiePathAuth = "/auth"
+	CookiePathRoot = "/"
+
+	MinPasswordLength = 8
+	MaxStringLength   = 128
```

Comment:
Al deze values moeten uit de env gehaald worden omdat ze later als configmaps of secrets geset moeten worden. Dit kunnen geen constants zijn omdat het cross-service concerns zijn en die wil je standardizen dmv configmaps en secrets, niet als constants in je code. Hier hebben we trouwens ook al wat oude values voor maar die moeten aangepast worden, zie de config.yaml files in de [platform-config repo](https://github.com/PTSS-Support/platform-config).

Reply:
Hmm, ik snap 	dat de meeste wel in config map kan, maar wat is de redenering voor AccessTokenDuration, RefreshTokenDuration? Waarom zou een andere service moeten weten hoelang het geldig is?

Reply:
Een andere service hoeft dat niet te weten, je kan in de platform-config repo je configmap application-specific (en environment-specific) maken. Het is makkelijkere management omdat ik de setup er al voor heb geschreven. Als het een constant is moet je bij een update de applicatie opnieuw opstarten, bij een configmap hoeft dat niet.

Reply:
maar je verliest versioning en sommige dingen worden nooit/ bijna niet aangepast. Snap niet de meerwaarde van de extra complexiteit wanneer het gaat over business logica. Dat zou je in de code zelf willen aanpassen, niet echt ergens anders lijkt me?
Denk dat het best is om ditte in constants te doen:
```
    AccessTokenCookie  = "access_token"     
    RefreshTokenCookie = "refresh_token"    
    MinPasswordLength = 8  
    accessTokenDuration: 900 
    refreshTokenDuration: 2592000 
```
en rest idd in de config. Dan eis je ook dat hieraan changes maken een pr vereist.

Reply:
Nou ja is niet echt business logic maar security logic. Verder, configmaps zijn versioned en hebben een PR nodig (mede door onze gitops approach in de platform-config). Dus met de constants approach verlies je alleen maar doordat je niet quick efficiency hebt. Ook kunnen we dan, indien nodig, verschil maken tussen dev en prod, voor testing. Verder, als we een frontend hadden die in openshift werd gehost, wilde je dit ook aan hun doorkunnen passen. Is in onze case niet zo maar laat wel blijken dat het meer future-proof is om dit in .env te doen.